requirements/development.py3.txt

```diff
@@ -1,6 +1,6 @@
 #
-# This file is autogenerated by pip-compile with python 3.6
-# To update, run:
+# This file is autogenerated by pip-compile with Python 3.11
```

Comment:
Hmm should I recompile the requirement's using 3.6?

Reply:
Not sure. We're due to deprecate 2.7 in the 1.0 release, but other than that we support "Python 3". I don't foresee any issues installing those reqs with Python 3.6, but it couldn't hurt to check. FWIW I use 3.6 because it's the most recently-eol'ed CPython, and just to be sure to be sure.

Reply:
Tried to regen using 3.6 (pyenv 3.6.15) think I am doing something wrong as it is the exact same. Even after deleting all existing requirement files and the `.venv`'s